### PR TITLE
ISPN-10533 Remove deprecated features hotrod-client-with-query and ho…

### PR DESCRIPTION
**Pull request by anistor**
…trod-client

https://issues.jboss.org/browse/ISPN-10533

### CODE REVIEW

**Comment by anistor** [comment]
Some leftovers remained. Weeding 'em all.

**Comment by danberindei** [comment]
Something seems wrong here, the build shouldn't have succeeded if the plugin was doing its job. And in the console output I see

```
[2019-09-08T11:38:28.744Z] 11:38:28.851 [INFO] --- karaf-maven-plugin:4.2.3:verify (validate) @ infinispan-client-hotrod ---
[2019-09-08T11:38:28.744Z] 11:38:28.853 [INFO] Using repositories: http://maven.repository.redhat.com/earlyaccess/all/@id=redhat-earlyaccess-repository-group@snapshots,https://repository.jboss.org/nexus/content/groups/public-jboss@id=jboss-public-repository-group@snapshots,https://repo.maven.apache.org/maven2@id=central
[2019-09-08T11:38:28.744Z] 11:38:28.897 [INFO] Features verified: 0, failures: 0, ignored: 0
```

both in master and in your PR's builds

File: client/hotrod-client/pom.xml [hunk 1]
```diff
@@ -603,7 +603,7 @@
             <artifactId>karaf-maven-plugin</artifactId>
             <configuration>
                <features>
-                  <feature>infinispan-client-hotrod,infinispan-client-hotrod-with-query,hotrod-client,hotrod-client-with-query</feature>
+                  <feature>infinispan-client-hotrod,infinispan-client-hotrod-with-query</feature>
```
Comment (danberindei) [inline client/hotrod-client/pom.xml]:
```suggestion
                  <feature>infinispan-client-hotrod</feature>
                  <feature>infinispan-client-hotrod-with-query</feature>
```

Comment (anistor) [reply client/hotrod-client/pom.xml]: fair enough

Comment (anistor) [reply client/hotrod-client/pom.xml]: I made that change. But did not get to investigate the CI failures as I'm trapped into something else. I saw them on master also, as you said.

**Comment by anistor** [comment]
Seems the failing tests ( org.infinispan.server.test.client.hotrod.HotRodCounterIT(distmode-udp).testCounters and org.infinispan.server.test.client.hotrod.HotRodCounterIT(replmodeudp).testCounters) are unrelated and happen also on master.  As for the karaf plugin doing its job, it seems it only verifies features that actually exist. A feature that does not exist is just ignored instead of being verified. Not very smart, but that's how it works, based on my tests. I'm merging this myself guys!

**Pull request merged**